docset/winserver2022-ps/appx/Get-AppxPackage.md

```diff
@@ -47,6 +47,33 @@ Get-AppxPackage -Name "Package17" -User "Contoso\EvanNarvaez"
 This command displays information about `Package17` if it's installed in the specified user
 profile.
 
+### Example 3: Get a particular app package information
+
+```powershell
+Get-AppxPackage -Name Microsoft.ScreenSketch
+```
+
+This command gives information about the ScreenSketch app.
+
+### Example 4: Get all app packages for a particular Publisher
+
+```powershell
+Get-AppxPackage -Publisher "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+```
+
+This command lists all the Microsoft app packages that are installed on the computer.
+
+### Example 5: Get all app packages with PackageTypeFilter
```

Comment:
According to the description of the parameter, we can specify with comma separated values like this, I tried and it works, so I think it is better to use that format

```
Get-AppxPackage -PackageTypeFilter bundle,framework,main,resource
```

Reply:
I think we should use the comma separated value example or is it any special reason to run the cmdlet 4 times? Thanks

Reply:
No special reason. They are all separated only. But users can choose any of those.

Reply:
@RAJU2529 
Hello - Andres/get-itips (and I) work with Microsoft. 
Please incorporate his suggestions, and then we can merge this PR.
Thank you!

Reply:
@tiburd  How do I reedit this article at present.

Reply:
@JohanFreelancer9 , please review send suggestions edit , so that I ca accept it

Reply:
Select the **Add a Suggestion** icon on the upper left of the comment box:
![image](https://github.com/MicrosoftDocs/windows-powershell-docs/assets/54688693/4280eb7a-11e5-4328-8c36-07f3905cb387)

Reply:
@JohanFreelancer9 
Please disregard the above comment. We'll take care of it.
Thanks!

Reply:
I think like this:
![image](https://github.com/MicrosoftDocs/windows-powershell-docs/assets/54688693/09e70acc-55ed-4652-b45f-dabad4dc0141)

And then for each of the following lines:
![image](https://github.com/MicrosoftDocs/windows-powershell-docs/assets/54688693/636279d8-df84-4b17-b889-f0972e7bed34)

Reply:
@RAJU2529 
If you have trouble, I will make the change. If you want me to do that, please provide the correct syntax for the comma-separated values.
Thank you!

Reply:
@tiburd , you mean starting word should be in Capital ?

Reply:
@RAJU2529 
Disregard. Thank you. I'll merge the PR after it passes the build checks.

Reply:
@tiburd . Thanking you.